src/components/overflow-menu/_overflow-menu.scss

```diff
@@ -145,7 +145,7 @@
   }
 
   .#{$prefix}--overflow-menu-options__option--danger {
-    .bx--overflow-menu-options__btn {
+    .#{$prefix}--overflow-menu-options__btn {
```

Comment:
I'm not sure if this has always been the case, but the overflow-menu doesn't show up in experimental mode

Reply:
Looks like this is related to product switcher? We can fix this when we build out the experimental overflow menu.
<img width="1290" alt="screen shot 2018-11-02 at 10 47 45 am" src="https://user-images.githubusercontent.com/2753488/47925723-daa78700-de8c-11e8-8fe3-7589db807ca1.png">

Reply:
Yeah I can work on that one next week and get it figured out 👍